musicadmin.html

```diff
@@ -133,6 +134,15 @@ <h6 v-if="card.title" class="card-title">{{ card.title }}</h6>
 							this.saving = false;
 						});
 					},
+					deleteCard (item) {
+					    axios
+						.get('', {
+							params: {
+								drop: this.tag_id
+							}
+						})
+						this.editing = false;
```

Comment:
this.editing = false; (and saving) MUST go in a `.then((response) => { /* here */ });`